src/object.d

```diff
@@ -424,6 +424,560 @@ class TypeInfo
     @property immutable(void)* rtInfo() nothrow pure const @safe @nogc { return rtinfoHasPointers; } // better safe than sorry
 }
 
+/*
+Generic implementation of TypeInfo. T must be unqualified.
```

Comment:
The docs say T must be unqualified, so why is there a need to call `Unqual!T` below?

Reply:
Comment got out of sync. Will fix, thanks.